### ISSUE TITLE
Unable to join server

### ISSUE THREAD

**Issue by tommasov03** [GeyserMC/Floodgate#563]
I have this problem, for a month now when a player tries to join my server via bedrock it makes him do an infinite loading and does not let him enter, then when he goes to press "cancel" on the loading of the connection to the server I receive the following logs on the Lobby and Gesyer consoles. Does anyone know how to help me?

Lobby log:
[16:20:35 INFO]: UUID of player .tommasov03 is 00000000-0000-0000-0009-01fa02c95a2e
[16:20:35 INFO]: [floodgate] Floodgate player logged in as .tommasov03 joined (UUID: 00000000-0000-0000-0009-01fa02c95a2e)
[16:20:45 INFO]: [floodgate] Floodgate player logged in as .tommasov03 disconnected
[16:20:45 INFO]: com.mojang.authlib.GameProfile@e5035be[id=00000000-0000-0000-0009-01fa02c95a2e,name=.tommasov03,properties={textures=[Property[name=textures, value=ewogICJ0aW1lc3RhbXAiIDogMTY5ODg1MzE5Mjk4MCwKICAicHJvZmlsZUlkIiA6ICIyZGM2MWIzYmZlZjM0YWNkOGU4MDZlNmFiMjdlMDkxYiIsCiAgInByb2ZpbGVOYW1lIiA6ICJ0b21tYXNvdjAzIiwKICAic2lnbmF0dXJlUmVxdWlyZWQiIDogdHJ1ZSwKICAidGV4dHVyZXMiIDogewogICAgIlNLSU4iIDogewogICAgICAidXJsIiA6ICJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlL2Y5ZWFkMzhkMjYzZDg3ODYzNGFjMGM5ZGYwYTU1OTNhOTM1YjE4M2JlZjUxNTdlMzlhN2Y5MjY1ODNjYTcxODEiCiAgICB9LAogICAgIkNBUEUiIDogewogICAgICAidXJsIiA6ICJodHRwOi8vdGV4dHVyZXMubWluZWNyYWZ0Lm5ldC90ZXh0dXJlL2Y5YTc2NTM3NjQ3OTg5ZjlhMGI2ZDAwMWUzMjBkYWM1OTFjMzU5ZTllNjFhMzFmNGNlMTFjODhmMjA3ZjBhZDQiCiAgICB9CiAgfQp9, signature=p7Nl2bKSmXuAv6UylG7SkQRyIUb4/EBp3//UsICs58VNccWPdrspRUlRGDaOvWgXQtAEaB66YLLDYZxKqPkJKBpv/8L510Nt94l6rYNNtJ367X2EJCabXdgfI5RzFCkUu29fl3qpXeqWafd2kDDuqsBpQ+t6Lv7TfT9DLjSLSo9uPAS5tk4cFcWmRXJ/Xuv8GtnXTGAjZeGvdhBDt4J8Amq7iYgNq4/N/paFgSEgutPn5KRVMRN8ako99OqoK8ZTNrmaM5GY0AMqhxs3x0YWcUk3kF2O7dPBF+XMfTZUBnvIgodU9BZx93orYAHUhbwhz4SfhRvEjgfof1XQTm5NKXOvER76h2TnQtAjCG1KHt/CoMYqe1FyxZqlVMPRgI6S8e2T6UiENVAd4mlaHiX3RJnagkmmJ5ayzMoZmkham2fi9Y54FME6GZi8Iz3UrRKaLhsi2BjpQKyCZjoeF5+dDe77c2yfFjRw0zmeIsQQJ/OE8KtYN6c4wEbdtt3Og5eg3nxW6TCTmXBtzwCSb4cmQ7o0vdmzuoEhb3zMpeQ6yWngxYysFt/HI6fjQgUtw+1f4KRxvnXhxv/mJ5EGM0GVIAwjXrBaagZln/aG7OjjajIK1jPpS0Wc/e+HmOa9YGIWJMDLOwM3287ulV4xm+IaBGNAp8fdA9mW3cZIyihJXAU=]]}] lost connection: Disconnected

Geyser log:
[16:20:35 INFO] Player connected with username tommasov03
[16:20:35 INFO] tommasov03 (logged in as: tommasov03) has connected to remote java server on address 172.18.0.1
[16:20:45 INFO] tommasov03 has disconnected from remote Java server on address 172.18.0.1 because of Bedrock client disconnected

This morning switching from using FlameCord to Waterfall gave me the same problems and the same logs except this additional one:

**[floodgate] Got disconnected from the skin uploader: failed to find the given code in combination with the verify code**

it is clear that the cause of the bedrock users not being connected is a problem with the skins.

**Comment by Tim203** [comment]
No, these are not related. For Geyser related support please use our Discord or look at our setup guide.
